templates/home.html

```diff
@@ -51,14 +51,8 @@ <h3 class="mb-4">Welcome<span v-cloak v-if="username"> back</span> to Wikipedia
 
         <div class="alert alert-warning" v-cloak role="alert" v-html="streakText" v-if="loggedIn && streakText"></div>
 
-        <p>The rules of this game are simple. You'll be given a starting Wikipedia article and an ending Wikipedia article. Your goal is to navigate from the starting article to the ending article using only the article links.</p>
-        <p>Some other things to keep in mind:</p>
-        <ul>
-            <li>Any link (outside the navigation bar at the top) is fair game.</li>
-            <li>You can't use the browser find tool.</li>
-            <li>Reloading the page or clicking the browser back button will end your run.</li>
-        </ul>
-        <p>Remember, luck is king, so put on your lucky hat, grab your lucky socks, and let the games begin!</p>
```

Comment:
:(  line 61 gave our site some personality but I'm fine deleting lol

Reply:
Yeah I really wanted to keep this line as well, but having just one sentence as a tag line does make it alot less cluttered. Will experiment with this a bit more.

Reply:
Opting to keep this line.